src/main/java/net/atos/client/zgw/zrc/model/RolNietNatuurlijkPersoon.java

```diff
@@ -44,7 +46,11 @@ public String getNaam() {
 
     @Override
     public String getIdentificatienummer() {
-        return getBetrokkeneIdentificatie() == null ? null : getBetrokkeneIdentificatie().getAnnIdentificatie();
+        return getBetrokkeneIdentificatie() != null
+                ? StringUtils.isNotEmpty(getBetrokkeneIdentificatie().getAnnIdentificatie())
+                ? getBetrokkeneIdentificatie().getAnnIdentificatie()
+                : getBetrokkeneIdentificatie().getInnNnpId()
+                : null;
```

Comment:
Zou die dubbele if-else-shorthand ook voorkomen kunnen worden door `?.` te gebruiken op `getBetrokkeneIdentificatie()`?

Reply:
Dit is Java, geen typescript.